lib/main.dart

```diff
@@ -33,7 +33,7 @@ class _RandomWordsState extends State<RandomWords> {
   final _suggestions = <WordPair>[];
   final _saved = <WordPair>{};
   final _biggerFont = const TextStyle(fontSize: 18.0);
-  bool _rowColor = false;
+  bool _followIconColor = false;
```

Comment:
The naming is still hard to understand. Consider name it something like `_changeLikeIconColor` or just `_changeIconColor` or `_toggleLikeIconColor`.

Reply:
You are right. I will rename)